Suggester docs test fails due to unexpected score

[This build](https://gradle-enterprise.elastic.co/s/uu3kdlan5ff4e) (https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob-unix-compatibility/os=fedora-29&&immutable/655/) failed as follows:

```
java.lang.AssertionError: Failure at [reference/search/suggesters/misc:64]: $body didn't match expected value:
                         $body: 
                         _shards: 
                            failed: same [0]
                           skipped: same [0]
                        successful: same [1]
                             total: same [1]
                            hits: 
                              hits: same [empty list]
                         max_score: same [null]
                             total: 
                            relation: same [eq]
                               value: same [0]
                         suggest: 
        phrase#my-second-suggester: 
                                   0: 
                                length: same [16]
                                offset: same [0]
                               options: 
                                       0: 
                                     score: expected Double [0.030227963] but was Double [0.039115205]
                                      text: same [some test message]
                                  text: same [some test mssage]
           term#my-first-suggester: 
                                   0: 
                                length: same [4]
                                offset: same [0]
                               options: same [empty list]
                                  text: same [some]
                                   1: 
                                length: same [4]
                                offset: same [5]
                               options: same [empty list]
                                  text: same [test]
                                   2: 
                                length: same [6]
                                offset: same [10]
                               options: 
                                       0: 
                                      freq: same [4]
                                     score: same [0.8333333]
                                      text: same [message]
                                  text: same [mssage]
                       timed_out: same [false]
                            took: same [18]
```

The `REPRODUCE WITH` line was as follows:

```
REPRODUCE WITH: ./gradlew ':docs:integTestRunner' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/search/suggesters/misc/line_10}" \
  -Dtests.seed=12A16F45234AB3D2 \
  -Dtests.security.manager=true \
  -Dtests.locale=ja-JP \
  -Dtests.timezone=America/Dawson \
  -Dcompiler.java=13
```

This **did** reproduce the failure the first time I tried it.

--- a/docs/reference/search/suggesters/misc.asciidoc
+++ b/docs/reference/search/suggesters/misc.asciidoc
@@ -26,7 +26,6 @@ POST _search?typed_keys
 }
 --------------------------------------------------
 // TEST[setup:twitter]
-// TEST[skip:awaits fix of https://github.com/elastic/elasticsearch/issues/54257]
 
 In the response, the suggester names will be changed to respectively `term#my-first-suggester` and
 `phrase#my-second-suggester`, reflecting the types of each suggestion:
@@ -79,6 +78,8 @@ In the response, the suggester names will be changed to respectively `term#my-fi
 }
 --------------------------------------------------
 // TESTRESPONSE[s/\.\.\./"took": "$body.took", "timed_out": false, "_shards": "$body._shards", "hits": "$body.hits"/]
+// TESTRESPONSE[s/"score": 0.8333333/"score": $body.suggest.term#my-first-suggester.2.options.0.score/]
+// TESTRESPONSE[s/"score": 0.030227963/"score": $body.suggest.phrase#my-second-suggester.0.options.0.score/]
 
 <1> The name `my-first-suggester` now contains the `term` prefix.
 <2> The name `my-second-suggester` now contains the `phrase` prefix.

